docs/meetings/team/2022_11_22.md

```diff
@@ -0,0 +1,84 @@
+# 🚀 Weekly Team Meeting Minutes
+
+**Start Time** 9:35am
+
+**End Time** 10:18am
+
+**Location** Discord
+
+**Date** Sept 16th 2022
+
+## 👋 Attendence
+
+1. Edouard
+2. Matt
+3. Veronica
+4. Logan
+
+***Attendence Comments***
+
+## 🧸 Check-In
+
+- "I only know 25 letters of the alphabet I don't y"
```

Comment:
```suggestion
- "I only know 25 letters of the alphabet I don't know y"
```

Reply:
somehow it doesn't have the same effect when written compared to spoken 😆